jest.config.js

```diff
@@ -1,4 +1,7 @@
-module.exports = {
+export default {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
+  },
```

Comment:
I copied this from https://kulshekhar.github.io/ts-jest/docs/guides/esm-support, but it isn't clear to me why this isn't included in any of the presets.